.travis.yml

```diff
@@ -7,6 +7,7 @@ services:
 env:
   - CRATE=.
   - CRATE=spaceapi
+  - CRATE=spaceapi_server
 script: cd $CRATE && cargo build --verbose && cargo test --verbose
```

Comment:
This is not Iron specific is it? You could commit it separately and directly into the rust branch.